daphne_worker/src/durable/reports_processed.rs

```diff
@@ -54,13 +56,24 @@ impl DurableObject for ReportsProcessed {
             state,
             env,
             touched: false,
+            alarmed: false,
         }
     }
 
     async fn fetch(&mut self, mut req: Request) -> Result<Response> {
         let id_hex = self.state.id().to_string();
         ensure_garbage_collected!(req, self, id_hex.clone(), BINDING_DAP_REPORTS_PROCESSED);
```

Comment:
This macro entails expensive DO->DO communication, which is solved by having the alarm do the garbage collection. We should be able to safely remove it, thereby saving latency substantially.

However, being able to clear all DO from storage via the `/internal/delete_all` endpoint is useful for testing. What do you think about fencing this for end-to-end testing only? We could inspect the value of `DAP_DEPLOYMENT` (see `config.rs`), and if the deployment type is not Prod, then we stick in the garbage collection queue.